roles/kubernetes-apps/ansible/templates/nodelocaldns-config.yml.j2

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nodelocaldns
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+
+data:
+  Corefile: |
+    {{ dns_domain }}:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+{% if secondaryclusterIP is defined %}
+        forward . {{ clusterIP }} {{ secondaryclusterIP }} {
+{% else %}
+        forward . {{ clusterIP }} {
+{% endif %}
+                force_tcp
+        }
+        prometheus :9253
+        health {{ local_dns_ip }}:8080
+        }
+    in-addr.arpa:53 {
+        errors
+        cache 30
+        reload
+        loop
+        bind {{ local_dns_ip }}
+{% if secondaryclusterIP is defined %}
```

Comment:
`secondaryclusterIP` will always be set. But you could add `dns_mode == 'coredns_dual'` to the check too